modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/soy/FileCard.java

```diff
@@ -24,10 +25,46 @@
  */
 public interface FileCard extends BaseClayCard {
```

Comment:
Could we extend `VerticalCard` instead of `BaseClayCard`? This would match the fact that `FileCardTag` extends `VerticalCardTag`. It would remove a lot of code duplication this is added in this PR.

The same question goes for `ImageCard`, we could have `ImageCard extends VerticalCard`.

Reply:
Yes of course, this is another thing I wasn't sure about but the change makes sense.

Reply:
If I understand correctly after looking at [`VerticalCard`](https://github.com/liferay-frontend/liferay-portal/blob/4482345d451c0503e038b62324b8b563d56810b8/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/soy/VerticalCard.java) and [`FileCard`](https://github.com/liferay-frontend/liferay-portal/blob/4482345d451c0503e038b62324b8b563d56810b8/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/soy/FileCard.java) (or [`ImageCard`](https://github.com/liferay-frontend/liferay-portal/blob/4482345d451c0503e038b62324b8b563d56810b8/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/soy/ImageCard.java)), it seems that `FileCard` and `ImageCard` wouldn't add anything else, is that correct?

I also need to fix a few things in [`VerticalCard`] that @carloslancha mentioned in his review.